composer.json

```diff
@@ -17,17 +17,18 @@
     ],
     "homepage": "https://github.com/php-http/throttle-plugin",
     "require": {
-        "php": ">=7.4",
-        "php-http/client-common": ">=1.3",
-        "symfony/rate-limiter": ">=5.2"
+        "php": "^7.4 || ^8.0",
+        "php-http/client-common": "^1.3 || ^2.0",
+        "symfony/rate-limiter": "^5.2 || ^6.0"
     },
     "require-dev": {
         "nyholm/psr7": "^1.0",
-        "php-http/mock-client": "*",
+        "php-http/mock-client": "^1.0",
         "ramsey/coding-standard": "^2.0",
-        "symfony/phpunit-bridge": ">=6.0"
+        "phpcsstandards/phpcsextra": "^1.0@alpha",
+        "phpcsstandards/phpcsutils": "^1.0@alpha",
+        "symfony/phpunit-bridge": "^6.0"
     },
-    "minimum-stability": "dev",
```

Comment:
I specifically want to test with unstable dependencies. I don't want to discover incompatibilities once library is released, that's when it's too late already.

Reply:
as long as you don't set up scheduled builds, we won't learn of upcoming incompatiblities.

the problem i have with this is that it can hide problems in the `require` section. for require-dev i don't mind installing with dev versions. but its no big issue for this small library, if you want to keep it this way.